.ci/.bump-stack-release-version.yml

```diff
@@ -49,3 +49,11 @@ projects:
     enabled: true
     labels: dependency
     title: "chore: update elastic stack release version"
+  - repo: e2e-testing
+    script: .ci/bump-stack-release-version.sh
+    branches:
+      - main
```

Comment:
This implementation does not use aliases, so we cannot use `8.<minor>` yet, so I'll hardcode this for the time being, then we can implement this feature, so I guess the last minor is `8.0` now?

```suggestion
      - "8.0"
```

Reply:
@mdelapenya , what do you think?

Reply:
These are the branches we currently support: main, 8.1, 8.0 and 7.17.

OTOH, there is no `.ci/bump-stack-release-version.sh` file in the repo, is this something we need to add?

Reply:
Yes, we need to do so, I guess the changes in https://github.com/elastic/e2e-testing/pull/2198 could be moved to the script `.ci/bump-stack-release-version.sh`

what do you think?